views/assets/stylesheets/components/code-reference.css

```diff
@@ -1,9 +1,11 @@
 .code-reference-container {
   display: flex;
   flex-direction: row;
+  color: --default-text-color;
```

Comment:
the value needs to be wrapped in the var function

Reply:
I've just removed, it's not needed